static/gallery/team.html

```diff
@@ -0,0 +1,623 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>NumPy</title>
+    <link rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+      crossorigin="anonymous">
+
+        <link rel="stylesheet"
+          href="/static/css/styles.css">
+
+    </head>
+    <body>
+      <header>
+      </header>
+      <main role="main">
+        <div class="container">
+          <h1 id="numpy-team-gallery">NumPy Team Gallery</h1>
```

Comment:
Wrong teams :)

Who do we add, all members of the SciPy org? Do we want to make teams like NumPy? Out of scope I know.

Reply:
Removed in 32433c9.   @stefanv is adding short codes and css for team galleries based on what he did for https://scientific-python.org.  I created #414 to remind us.